keras_cv/bounding_box/utils.py

```diff
@@ -63,8 +63,9 @@ def clip_to_image(bounding_boxes, images, bounding_box_format):
 
     Args:
         bounding_boxes: bounding box tensor to clip.
-        images: list of images to clip the bounding boxes to.
         bounding_box_format: the KerasCV bounding box format the bounding boxes are in.
+        images: list of images to clip the bounding boxes to.
```

Comment:
Maybe we should throw if both images and image_shape are None? (Or maybe that automatically happens elsewhere?)

Reply:
Happens in convert_format if needed